README.md

```diff
@@ -399,6 +400,24 @@ render((
         uiSchema={uiSchema} />
 ), document.getElementById("app"));
 ```
+### Array items ordering
+
+Array items are sortable by default. The `uiSchema` object spec allows you to disable ordering:
+
+```jsx
+const schema = {
+  type: "array",
+  properties: {
+    type: "string"
+  }
+};
+
+const uiSchema = {
+  "ui:options":  {
+    sortable: false
+  }
```

Comment:
Having the same approach as widgets would be more consistent, but the `ui:options` -style makes updating a `uiSchema` simpler (if you want to add an option to an existing uiSchema field, you don't replace any lines, just add new ones).